spec/druid_list_xsd_spec.rb

```diff
@@ -55,7 +55,7 @@
         druidlist.validate(xml).each do |error|
           errors.push error.message
         end
-        expect(errors.size).to eq 2
+        expect(errors.size).to eq 1
```

Comment:
This is sketchy, but not sure why this was previously `2`

Reply:
weird.  i checked out a few older commits, and the specs fail on the same line as the one changed here.

the value in this PR does make more sense to me.  i `puts`ed the errors when i ran tests on the older branches as well as current `master`, and in all cases the errors array before the `expect` looked something like `["1:0: ERROR: Element 'druid': [facet 'pattern'] The value 'wj852 5z419' is not accepted by the pattern '[a-zA-Z]{2}[0-9]{3}[a-zA-Z]{2}[0-9]{4}'."]`

`¯\_(ツ)_/¯`

hard to say when it last worked, i'm fine just going with this, since it seems correct.